.github/CODEOWNERS

```diff
@@ -2,11 +2,11 @@
 # the repo. Unless a later match takes precedence
 *   @sosey
 
-/notebooks/ @drlaw1558  @sosey
+/notebooks/ @drlaw1558  @rizeladiaz
 
 # Order is important; the last matching pattern takes the most
 # precedence.
-/MIRI/   @skendrew  @drlaw1558
-/NIRISS/   @spacetelescope/niriss  @drlaw1558
-/NIRSPEC/   @spacetelescope/nirspec  @drlaw1558
-/NIRCAM/   @spacetelescope/nircam  @drlaw1558
+/notebooks/MIRI/   @skendrew  @drlaw1558
+/notebooks/NIRISS/Imaging/   @spacetelescope/niriss  @drlaw1558
+/NIRSPEC/Slit/   @spacetelescope/nirspec  @drlaw1558
```

Comment:
Presumably NIRSPEC and NIRCAM also need notebooks/ in the path?

Reply:
doh. friday night commit - thanks